### PR TITLE
V2 k8s updates

### PR DESCRIPTION
**Pull request by bascar**
Updating Katacoda scenario for k8s fir PX-Enterprise 2.0

### CODE REVIEW

File: deploy-px-k8s/etcd.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ docker run -d --net=host -p 4001:2379 \
    --initial-cluster node1=http://${IP}:2380
 ```{{execute T1}}
 
-For production setups, follow [this page](https://docs.portworx.com/maintain/etcd.html) for recommendations on etcd.
+For production setups, follow [Portworx docs](https://docs.portworx.com/) for the recommendations on etcd. As of Version 2.0 you have the option to use a built in etcd.
```
Comment (harsh-px) [inline deploy-px-k8s/etcd.md]: @bascar Update the newly added etcd doc link here.

**Comment by BenHall** [comment]
Looking forward to hearing more about this! Is the plan to have both 1.5
content and V2 on the site for the time being?

File: deploy-px-k8s/etcd.md (hunk 1 above)
Comment (bascar) [reply deploy-px-k8s/etcd.md]: Updated

File: deploy-px-k8s/install-px.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ VER=`kubectl version --short | awk -Fv '/Server Version: /{print $3}'`
 curl -L -s -o px-spec.yaml "http://install.portworx.com/1.4?c=px-demo&k=etcd://master:4001&kbVer=${VER}&s=/dev/vdb"
 ```{{execute T1}}
 
-Observe how we used [query parameters](https://docs.portworx.com/scheduler/kubernetes/install.html#generating-the-spec) in the curl command to customize our spec.
+Observe how we used [query parameters](https://docs.portworx.com/portworx-install-with-kubernetes/on-premise/other/#) in the curl command to customize our spec.
```
Comment (harsh-px) [inline deploy-px-k8s/install-px.md]: this URL should be https://docs.portworx.com/portworx-install-with-kubernetes/px-k8s-spec-curl/

**Pull request closed without merging**